wfl/App_Start/OwinConfiguration.cs

```diff
@@ -12,9 +15,15 @@ public class OwinConfiguration
     {
         public void Configuration(IAppBuilder app)
         {
-            HttpConfiguration configuration = new HttpConfiguration();
+            IUnityContainer container = new UnityContainer();
+            DIContainerConfig.RegisterElements(container);
+            container.RegisterType(typeof(RestaurantsController));
```

Comment:
Can we choose either to register types here, or register them in the DIContainerConfig?  It feels strange to split the configuration like this.